Src/WitsmlExplorer.Api/Jobs/Common/MessageObjectReference.cs

```diff
@@ -0,0 +1,9 @@
+namespace WitsmlExplorer.Api.Jobs.Common
+{
+    public class MessageObjectReference
+    {
+        public string Uid { get; set; }
```

Comment:
A suggestion to try from now on is to use `{ get; init; }` instead of `{ get; set; }`. Using init makes the classes immutable after initialization which makes sense as data objects. This is an overall comment not just for this code line and which perhaps could be a subject for a project discussion about coding style

Reply:
I like this idea 👍